server/src/main/java/org/apache/iotdb/db/qp/physical/PhysicalPlan.java

```diff
@@ -170,7 +175,7 @@ public void serialize(ByteBuffer buffer) {
    *
    * @param buffer
    */
-  public void deserialize(ByteBuffer buffer) throws IllegalPathException {
+  public void deserialize(ByteBuffer buffer) throws IllegalPathException, IOException {
```

Comment:
![image](https://user-images.githubusercontent.com/30497621/127079991-1b0a08ac-dd3e-4058-b834-2a9cd6faca89.png)

IOException may be thrown by `SelectIntoPlan#deserialize`. To be specific, it may be thrown by `queryPlan = (QueryPlan) Factory.create(buffer);`